inc/Addon/GoogleTracking/Subscriber.php

```diff
@@ -13,7 +13,8 @@
  * @since 3.1
  * @author Remy Perona
```

Comment:
Remove all `@author` instances in this class.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/dd041589183f9e6ea56fdbce81270869d5f90c2e